### PR TITLE
💄 Update icon button styles

### PR DESCRIPTION
**Pull request by alyssadsouza**
Some UI refactoring/cleanup before adding the consent card to the Participant Dashboard:
- Removed the `layout` prop and replaced with optional `LeftIcon` and `RightIcon` props that accept an Icon component and support left and right icon styling (different padding/position)
- Added an optional `action` prop with options `"next" | "prev"` for special right/left arrow icon buttons
- Updated the padding and font size for all buttons

**Button Previews**
`base`
Screenshot 2023-11-06 at 11 40 32 AM
`large`
Screenshot 2023-11-06 at 11 44 18 AM
`next`, `base` (and `RightIcon`)
Screenshot 2023-11-06 at 11 33 24 AM
`prev`, `base` (and `LeftIcon`)
Screenshot 2023-11-06 at 11 33 34 AM
`next`, `large` (and `RightIcon`)
Screenshot 2023-11-06 at 11 35 50 AM
`prev`, `large` (and `LeftIcon`)
Screenshot 2023-11-06 at 11 37 08 AM

### CODE REVIEW

File: apps/consent-ui/src/components/Button/Button.module.scss [hunk 1]
```diff
@@ -103,13 +105,39 @@
...
+    padding-left: 0.5rem;
+  }
+  &.large {
+    padding-right: 2rem;
+    padding-left: 1.2rem;
+  }
+}
+
+.left-arrow {
+  transform: rotate(180deg);
```
Comment (samrichca) [inline apps/consent-ui/src/components/Button/Button.module.scss:142]: the 🚫 at the bottom of this file - is prettier working? 🤔

Comment (alyssadsouza) [reply apps/consent-ui/src/components/Button/Button.module.scss:142]: Ah I forgot to update these, I'll do that now

File: apps/consent-ui/src/components/Header/Header.module.scss [hunk 2]
```diff
@@ -26,15 +26,15 @@
 	align-items: center;
 }
 
-.help {
+.header .help {
```
Comment (samrichca) [inline apps/consent-ui/src/components/Header/Header.module.scss:29]: `.help` can be nested under `.header`, also `.helpButton padding-left: 0.3rem` makes the left padding more even with the top & bottom padding  

image 

image

Comment (alyssadsouza) [reply apps/consent-ui/src/components/Header/Header.module.scss:29]: the mockups make it look very close to the edge though 🤔 
image

Comment (samrichca) [reply apps/consent-ui/src/components/Header/Header.module.scss:29]: true true. i thought the bigger vertical padding was on purpose.

the icon looks a little bigger in the mockup, relative to the text. maybe make it bigger and closer to the border

Comment (samrichca) [reply apps/consent-ui/src/components/Header/Header.module.scss:29]: the right padding specifically on the help button is also about 25% smaller than on standard buttons

other than that and the shrinking chevron everything looks perfect 🔍 👀

**Comment by samrichca** [comment]
lgtm! 🚀

**Review by samrichca — APPROVED** [review]

**Review by samrichca — APPROVED** [review]

**Pull request merged**